src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -222,13 +222,12 @@ function MultiCombobox<Item>({
             onMouseDown: toggleMenu,
             readOnly: true,
             placeholder: !value.length ? placeholder : '',
-            position: !value.length ? 'static' : 'absolute',
           }),
           ...(searchable && {
-            placeholder,
-            mt: (isOpen && value.length) || (isOpen && value.length && allowAdditions) ? -4 : 0,
-            position: isOpen || (isOpen && value.length && allowAdditions) ? 'static' : 'absolute',
+            placeholder: hideLabel && value.length && !isOpen ? '' : placeholder,
```

Comment:
is it easy to ellipsify the placeholder? I acn see the placeholder text being cut off sometimes.

See [here](https://codepen.io/bradleytaunt/pen/OgpzyY/) for an example. We can also just remove the placeholder when **at least one value** has been added (i.e. only show it when no values are present)